MediaWikiTelegramBot.js

```diff
@@ -284,7 +293,38 @@ function showCurrentMwMessage(userID) {
 function advanceMwMessage(userID) {
     const user = getUser(userID);
 
+    // Make sure there is another message to translate
+    if( user.currentMwMessageIndex + 1 >= user.loadedMwMessages.length ) {
+        user.currentMwMessageIndex = 0;
+        user.loadedMwMessages = [];
+
+        // Prepare the fetch untranslated button
+        const inlineKeyboard = [inlineKeyboardButton(
```

Comment:
The creation of a "load untranslated" button could probably be moved to a function.